doc/leaderf.txt

```diff
@@ -986,6 +986,12 @@ USAGE                                           *leaderf-usage*
 :LeaderfWindow                                  *LeaderfWindow*
     Launch LeaderF to search windows.
 
+:LeaderfQuickFix                                *LeaderfQuickFix*
```

Comment:
Seems these two commands are not defined yet.

Reply:
I'm sorry.
Added.